src/website/views/components/tabs/_examples.pug

```diff
@@ -1058,7 +1058,7 @@ h2 Examples
 
         <script>chi.tab(document.getElementById('example-tabs-2'));</script>
 
-.-portal--hide
+div
```

Comment:
```suggestion
```

Reply:
@kosgor, div tag is not neccesary in pug files. Code asociated with Tabbed navigation bar must be indented properly. You can take a look at close icon button in button doc page to see how it is done.  
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/17043903/225584996-b513b659-f0b4-4ca8-b9b3-eeb4f2a37d2e.png)